src/py/mat3ra/made/tools/build/interface/builders.py

```diff
@@ -62,7 +62,8 @@ def _update_material_name(self, material: Material, configuration: InterfaceConf
 #                           Simple Interface Builder                                   #
 ########################################################################################
 class SimpleInterfaceBuilderParameters(InterfaceBuilderParameters):
-    scale_film: bool = True
+    scale_film: bool = True  # Whether to scale the film to match the substrate
+    slab_as_is: bool = False  # Whether to use the substrate as is as a slab instead of creating a slab
```

Comment:
Let's use `create_slabs`, and have it as True by default

Reply:
Here I use this only for substrate. But we can adjust to be for both, but then we would have to create a slab before passing it